api/v1alpha1/project.go

```diff
@@ -17,6 +19,12 @@ type ProjectSpec struct {
 	// Owners is the list of identifiers of owners of the project.
 	// It can be any strings that uniquely identifies the owners, such as email addresses or Jac group identifiers.
 	Owners []string `yaml:"owners,omitempty" json:"owners,omitempty"`
+
+	// Git repository of the project.
+	Repository string `yaml:"repository,omitempty" json:"repository,omitempty"`
+
+	// Location of the project in the repository. Should be empty if the whole repository is the project.
+	RepositoryPath string `yaml:"repositoryPath,omitempty" json:"repositoryPath,omitempty"`
```

Comment:
Does anybody have a better name for this?

```suggestion
	PathInRepository string `yaml:"repositoryPath,omitempty" json:"repositoryPath,omitempty"`
```

Reply:
Subdirectory maybe?

Reply:
`RepositorySubDirectory`?